### PR TITLE
Views

### PR DESCRIPTION
**Pull request by yashodhicy**
Implement the design from the sneak peek wireframes.
output
image
image
image
image

### CODE REVIEW

File: app/views/users/_user.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+</header>
+<body>
+<div id="user"> 
+  <img id="user-image" src="<%= user.photo %>" alt="User Photo">
+  <article id="user-details">
+    <h1><%= user.name %></h1>
+    <p>Number of posts: <%= user.posts_counter.nil? ? 0 : user.posts_counter %></p>
+  <article>
+</div>
+</body>
```
Comment (KanzaTahreem) [inline app/views/users/_user.html.erb:12]: _Well done for creating partials 🎉✨_

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @yashodhicy 

Good job so far 👏! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Great job with CSS
- ✅ Correct Linters

## Required Changes 🛑
- [x] Please ensure that we receive the user's index page when the app starts. Currently, Rails renders the default page. Please refer to the URL in the wireframe below 👇. You can achieve this by setting `root users#index` in `routes.rb`.

  image
  
  _Kindly do not edit the current implementation of routes, you can simply add a root. There's no need to edit routes for this_
  
_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/_user.html.erb (hunk 1 above)
Comment (yashodhicy) [reply app/views/users/_user.html.erb:12]: Thank you

**Review by DammyShittu — APPROVED** [review]
Hi @yashodhicy,

## APPROVED 🟢

## Highlights
✔️ Project met all technical requirements

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**